src/Raven.Server/Rachis/Follower.cs

```diff
@@ -297,23 +294,23 @@ private async Task Concurrent_SendAppendEntriesPendingToLeaderAsync(Cancellation
                     }
                 }
 
-                lastLogIndex = _engine.GetLastEntryIndex(context);
-
                 var lastEntryIndexToCommit = Math.Min(
-                    lastLogIndex,
+                    _engine.GetLastEntryIndex(context),
```

Comment:
Shouldn't we say, my last index entry or (if you gave me any entries, the last entry index you gave me) or the leader commit?